keploy/keploy.go

```diff
@@ -339,11 +340,40 @@ func (k *Keploy) check(runId string, tc models.TestCase) bool {
 	return false
 }
 
+func (k *Keploy) getHeaderFilter(tcs regression.TestCaseReq) bool {
```

Comment:
@gouravkrosx this method name doesn't sound right. `getHeaderFilter` means it should return a `HeaderFilter`, but its returning a `bool`

Reply:
Sure, i will rename that